server/sources.go

```diff
@@ -328,6 +330,8 @@ func (s *Service) HandleNewSources(ctx context.Context, input string) error {
 		return nil
 	}
 
+	s.Logger.Error("--new-sources is depracated. To preconfigure a source, see this link. www.example.com")
```

Comment:
Should this say `www.example.com`, @russorat ?

Reply:
filed https://github.com/influxdata/chronograf/issues/2854 to address